### PR TITLE
Adding computer readable time

### PR DESCRIPTION
**Pull request by adhishmeena**
#### Does this PR introduce a user-facing change?

```release-note
Adding computer readable time in response to cmd => podman events --format "{{json .}}"
```
Adding computer readable time in response to cmd=> podman events --format "{{json .}}"

### CODE REVIEW

**Comment by rhatdan** [comment]
Thanks @adhishmeena You need to sign your commits

git commit -a --amend -s
git push --force

**Comment by rhatdan** [comment]
@mheon PTAL

File: pkg/domain/entities/events.go [hunk 1]
```diff
@@ -74,7 +74,7 @@ func ConvertToEntitiesEvent(e libpodEvents.Event) *Event {
 		},
 		Scope:    "local",
 		Time:     e.Time.Unix(),
-		TimeNano: e.Time.UnixNano(),
```
Comment (giuseppe) [inline pkg/domain/entities/events.go:77]: should this chunk be reverted?

Comment (adhishmeena) [reply pkg/domain/entities/events.go:77]: yes ,Please revert this.

**Comment by vrothberg** [comment]
Friendly ping

**Comment by adhishmeena** [comment]
Can you please share doc for local setup of podman?

**Comment by rhatdan** [comment]
You need to gofmt your code.

**Comment by baude** [comment]
Ci wants you to add a test.  Please consider doing so.

**Comment by adhishmeena** [comment]
@baude , Would you please guide me , how to add test for CI .

**Comment by TomSweeneyRedHat** [comment]
@adhishmeena I think what @baude is looking for is a test similar to https://github.com/containers/podman/blob/main/test/e2e/events_test.go#L53 where the date format is verified.  HTH

**Comment by vrothberg** [comment]
Friendly ping, @adhishmeena.  Do you have time to kick this PR over the finish line?

**Comment by vrothberg** [comment]
Closing due to inactivity.  Feel free to reopen when you find time.

**Pull request closed without merging**